### ISSUE TITLE
[BE] First Backend

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#21]
Create API Endpoint that accepts text from the Front end

## Acceptance criteria
* The Endpoint is callable and accepts the text -> Text is shown somewhere (e.g. Console)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
